app/services/subscription.js

```diff
@@ -118,6 +120,11 @@ export default Service.extend(Evented, {
   },
 
   wire() {
+    if (this.isSubscribed) {
+      return;
+    }
+
+    this.isSubscribed = true;
```

Comment:
@swatijadhav Looks good, just don't forget to set it back to false in the `unwire()` method